src/cockpit/channels/dbus.py

```diff
@@ -135,9 +135,9 @@ async def do_call(self, call, message):
         # We have to figure out the signature of the call.  Either we got told it:
         signature = message.get('type')
 
-        # ... or all of our arguments are string (and we can guess):
```

Comment:
This was meant to be a reasonable default in case the signature was not explicitly specified.